rdf-star-cg-spec.html

```diff
@@ -873,7 +873,7 @@ <h2>SA-mode and PG-mode</h2>
 
       <ul>
         <li>The <code>&lt;&lt; ... &gt;&gt;</code> syntax represents an <a>embedded triple</a> without asserting it, satisfying the need formerly filled by SA-mode.</li>
-        <li>The <code>{| ... |}</code> syntax creates triples where the subject is interpreted as an <a>embedded</a> version of the triple previously requiring distinct <a>assertion</a>, without the need to repeat it, satisfying the need formerly filled by PG-mode.</li>
+        <li>The <code>{| ... |}</code> syntax (annotation) creates triples where the subject is interpreted as an <a>embedded</a> version of the triple <a>asserted</a> just before the annotation, without the need to repeat it, satisfying the need formerly filled by PG-mode.</li>
```

Comment:
```suggestion
        <li>The <code>{| ... |}</code> annotation syntax creates triples where the subject is interpreted as an <a>embedded</a> version of the triple <a>asserted</a> just before the annotation, without the need to repeat it (i.e., to explicitly assert it), satisfying the need formerly filled by PG-mode.</li>
```

Reply:
Exactly what's trying to be said is not 100% clear, so I'm not sure this says it ... but I think it's closer than it started.

Reply:
No sorry, I think we are getting further from what I'm trying to say (but this is cumbersome to put in prose).
```
:a :b :c {| :p1 :o1; :p2 :o2 |}
```
The annotation is the `{| ... |}` part. The triple `:a :b :c` is asserted, as it would be if there was no annotation. What the annotation is doing is creating triples `X :p1 :o1` and `X :p2 :o2`, where X is `<< :a :b :c >>`, an embedded version of `:a :b :c`, the triple asserted just before the annotation.

I think the "is interpreted as" is confusing and should be removed. I also agree that getting "annotation" out of parenthesis is better. Finally, I'll add `:p :o` inside the annotation syntax example, to hint at the fact that it contains predicate-object pairs, but no explicit subject.

Reply:
@TallTed what do you think now?